components/DetailsPageButton/DetailsPageButton.styled.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export const StyledButton = styled.button`
+height: 5vh;
```

Comment:
I wouldn't recommend using `vh` for the height of the button, because it's based on the `viewport's height` of the user and could squeeze the button:
![screenshot](https://github.com/Airglow98/Capstone-Project/assets/82408272/469d9d95-4892-4dc6-886f-2f0b2345c8c5)
Instead you could use `padding` for example to get spacing around the text of the button (`padding: 10px 0;`)